src/lib/docs.ts

```diff
@@ -90,17 +92,28 @@ const serializeSignature = ({
   checker: TypeChecker;
   signature: Signature;
 }): DocEntryConstructor => {
-  return {
+  const result: DocEntryConstructor = {
     parameters: signature.parameters.map((symbol: TypeScriptSymbol) =>
       serializeSymbol({checker, symbol})
     ),
     returnType: checker.typeToString(signature.getReturnType()),
     documentation: displayPartsToString(signature.getDocumentationComment(checker)),
-    visibility:
-      signature.declaration?.modifiers?.[0].kind === SyntaxKind.PrivateKeyword
-        ? 'private'
-        : 'public'
+    visibility: 'public'
   };
+
+  if (!signature.declaration) {
```

Comment:
What do you think about a single if?

```
if (signature.delation !== undefined && 'modifiers' in signature.declaration) {
   return {
     ....
   }
  }

  return result;
};

Reply:
this is better. late night coding brain does some weird things. will update.